src/include/gcc.h

```diff
@@ -164,16 +164,8 @@ WT_ATOMIC_FUNC(size, size_t, size_t *vp, size_t v)
     do {                                         \
         __asm__ volatile("mfence" ::: "memory"); \
     } while (0)
-#define WT_READ_BARRIER()                        \
-    do {                                         \
-        __asm__ volatile("lfence" ::: "memory"); \
-    } while (0)
-/* We only need a compiler barrier for x86 as its memory ordering is strong enough. */
-#define WT_READ_BARRIER_WEAK_MEMORDER() WT_BARRIER()
-#define WT_WRITE_BARRIER()                       \
-    do {                                         \
-        __asm__ volatile("sfence" ::: "memory"); \
-    } while (0)
+#define WT_READ_BARRIER() WT_BARRIER()
```

Comment:
```suggestion
/* We only need compiler barriers on x86 as Total Store Ordering (TSO) guarantees StoreStore and LoadLoad ordering. */
#define WT_READ_BARRIER() WT_BARRIER()
```

Reply:
We'll need to change this to release / acquire when we get that.